### ISSUE TITLE
picard_MergeSamFiles and sam_sort appear to be performing the same operation

### ISSUE THREAD

**Issue by jennaj** [galaxyproject/tools-devteam#162]
Duplication is desired? This may be confusing for end users. Both samtools and picard are nearly always installed in locals as the first go-to tool packages.

**Comment by bgruening** [comment]
Do you really mean `MergeSamFiles` and `sam_sort`?
I don't think duplication is desired and we should demultiplex tools into it's own repository, offering more suites that fits specific needs.

**Comment by jennaj** [comment]
Yes, both of these. And I like that idea. We pulled the problematic MergeSAMFiles from the upcoming cloudman build. The sam_sort is only problematic for the single tag and is across all tools I have tested that include it. It is not on Main, just MTS and now the cloudman build (for now).

**Comment by bgruening** [comment]
@jennaj I don't see were [MergeSamFile](https://github.com/galaxyproject/tools-devteam/blob/master/tools/picard/picard_MergeSamFiles.xml) do the same than [sam_sort](https://github.com/galaxyproject/tools-devteam/blob/master/tool_collections/samtools/samtools_sort/samtools_sort.xml). The one is merging multiple BAM files and can accept multiple inputs the other one accepts one input and sorts it.

Do you mean the specific option: `Assume the input file is already sorted`?
This can be removed I guess. If you are referring to this let me know and I will patch it.

**Comment by jennaj** [comment]
Ok, there are two duplicated tool pairs. This is one of them. I'll open another ticket for the BAM/SAM merging function duplicated tools. Sorry - many tools in play on the cloud build and some had issues.

The "Assume the input file is already sorted" issue is also distinct. Removing it is one option. If we go that way, mentioning in the usage on the tool form to use one of these sort tools (whichever we retain) would be very helpful for users.
